client/my-sites/checkout/src/payment-methods/credit-card/credit-card-number-field.tsx

```diff
@@ -79,7 +79,9 @@ export default function CreditCardNumberField( {
 						handleStripeFieldChange( input );
 					} }
 				/>
-				<PaymentLogo brand={ brand } />
+				{ hasCheckoutVersion( 'cobadge' ) && (
+					<PaymentLogo brand="Added some cobadge related changes!" />
+				) }
```

Comment:
The string passed to hasCheckoutVersion is what you'll use as the URL query parameter `/checkout?checkoutVersion='cobadge'`